lib/ansible/modules/network/edgeos/edgeos_command.py

```diff
@@ -95,40 +95,30 @@
   type: list
   sample: [['...', '...'], ['...'], ['...']]
 """
-
 import time
 
 from ansible.module_utils._text import to_text
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.network.common.parsing import Conditional
-from ansible.module_utils.network.common.utils import ComplexList
+from ansible.module_utils.network.common.utils import ComplexList, to_lines
 from ansible.module_utils.network.edgeos.edgeos import run_commands
-from ansible.module_utils.six import string_types
-
-
-def to_lines(stdout):
-    for item in stdout:
-        if isinstance(item, string_types):
-            item = to_text(item).split('\n')
-        yield item
 
 
 def parse_commands(module, warnings):
-    spec = dict(
+    transform = ComplexList(dict(
```

Comment:
Can the logic of creating ComplexList object moved to a common location with the updated spec. This change can add support for multiple prompts and multiple answers for platform-specific command modules without any code change in the module (doc might be required to be updated)


```
    transform = ComplexList(dict(
        command=dict(key=True),
        prompt=dict(type='list'),
        answer=dict(type='list'),
        sendonly=dict(type='bool', default=False),
        check_all=dict(type='bool', default=False),
    ), module)
    commands = transform(module.params['commands'])
```

Reply:
Yeah, this could probably be centralized. The thing I'm not sure about is that only some of the modules want `output` as a key and I'm not sure if including that will introduce problems.

It seems like it shouldn't be a problem; I don't think it should create unnecessary keys, but I'll have to look into it.